src/qt/transactionview.h

```diff
@@ -112,8 +116,8 @@ public Q_SLOTS:
     void chooseDate(int idx);
     void chooseType(int idx);
     void chooseWatchonly(int idx);
-    void changedPrefix(const QString &prefix);
-    void changedAmount(const QString &amount);
+    void changedPrefix();
```

Comment:
can be moved to the private Q_SLOTS section. All the others can as well, mabye in a separate commit?